tests_metricflow/fixtures/source_table_snapshots/time_spine_table/mf_time_spine_microsecond.yaml

```diff
@@ -15,23 +15,23 @@ table_snapshot:
     - ["2020-01-01 00:00:00.000008"]
     - ["2020-01-01 00:00:00.000009"]
     - ["2020-01-01 00:00:00.000010"]
-    - ["2020-01-01 00:00:00.000011"]
-    - ["2020-01-01 00:00:00.000012"]
-    - ["2020-01-01 00:00:00.000013"]
-    - ["2020-01-01 00:00:00.000014"]
-    - ["2020-01-01 00:00:00.000015"]
-    - ["2020-01-01 00:00:00.000016"]
-    - ["2020-01-01 00:00:00.000017"]
-    - ["2020-01-01 00:00:00.000018"]
-    - ["2020-01-01 00:00:00.000019"]
-    - ["2020-01-01 00:00:00.000020"]
-    - ["2020-01-01 00:00:00.000021"]
-    - ["2020-01-01 00:00:00.000022"]
-    - ["2020-01-01 00:00:00.000023"]
-    - ["2020-01-01 00:00:00.000024"]
-    - ["2020-01-01 00:00:00.000025"]
-    - ["2020-01-01 00:00:00.000026"]
-    - ["2020-01-01 00:00:00.000027"]
-    - ["2020-01-01 00:00:00.000028"]
-    - ["2020-01-01 00:00:00.000029"]
-    - ["2020-01-01 00:00:00.000030"]
+    - ["2020-01-02 00:00:00.000011"]
+    - ["2020-01-02 00:00:00.000012"]
+    - ["2020-01-02 00:00:00.000013"]
+    - ["2020-01-02 00:00:00.000014"]
+    - ["2020-01-02 00:00:00.000015"]
+    - ["2020-01-02 00:00:00.000016"]
+    - ["2020-01-02 00:00:00.000017"]
+    - ["2020-01-02 00:00:00.000018"]
+    - ["2020-01-02 00:00:00.000019"]
+    - ["2020-01-02 00:00:00.000020"]
+    - ["2020-01-03 00:00:00.000021"]
```

Comment:
Argh I just realized these will no longer be contiguous, but I think that's fine. I guess we could do a midnight span but eh....